torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12643,7 +12643,8 @@ def error_inputs_mean(op_info, device, **kwargs):
            sample_inputs_func=sample_inputs_max_min_reduction_with_dim,
            supports_fwgrad_bwgrad=True,
            skips=(
-               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_non_standard_bool_values'),
+               DecorateInfo(unittest.skip("fails on some platforms"),
```

Comment:
Ideally we would skip this test on just those platforms, since this test seems important for this PR

Reply:
@mruberry changed to just disable the xfail on macos-x86.